### DISCUSSION TITLE
Make worldPos() and screenPos() accessible

### DISCUSSION THREAD

**Discussion by amyspark-ng** [kaplayjs/kaplay#22] [Ideas]
Maybe rework the pos system? Since having 3 different pos may be confusing, maybe not much if the names were changed

**Comment by mflerackers** [comment]
pos is your local position relative to your parent (aka translation)
worldPos is your position relative to the root (note that rotation and scale of ancestors can influence this too, not just their pos)
screenPos is your position on the screen. It is your worldPos transformed by the camera.
